src/app/dataset/dataset-create/dataset-create.component.html

```diff
@@ -18,11 +18,28 @@ <h4 class="display-4">Create a dataset</h4>
              [(ngModel)]="dataset.description">
     </div>
 
+    <!-- Dataset content -->
+    <div class="form-group">
+      <label class="control-label" for="inputFile">CSV</label>
+      <div class="col-md-9">
+        <input type="file" id="inputFile" autofocus="" (change)="addDataFile($event)" #file/>
+      </div>
+
+    </div>
+
+    <!-- Dataset separador -->
+    <div class="form-group">
```

Comment:
It is better to not let users define this value because it depends on the file that has been uploaded. You can get the content type of the file using `File.type`, more details:
https://developer.mozilla.org/es/docs/Web/API/File/type